internal/mutexkv.go

```diff
@@ -1,4 +1,4 @@
-package equinix
+package internal
```

Comment:
This move also seems to not impact the SSH key refactor and could be removed from this PR without waiting for a separate mutexkv refactor PR to be merged.